docs/apache-airflow/administration-and-deployment/logging-monitoring/advanced-logging-configuration.rst

```diff
@@ -30,7 +30,7 @@ can also customize it and configure it as you want by overriding Python logger c
 be configured by providing custom logging configuration object. Some configuration options require
 that the logging config class be overwritten. You can do it by copying the default
 configuration of Airflow and modifying it to suit your needs. The default configuration can be seen in the
-`airflow_local_settings.py template <https://github.com/apache/airflow/blob/|airflow_version|/airflow/config_templates/airflow_local_settings.py>`_
+`airflow_local_settings.py template <https://github.com/apache/airflow/blob/main/airflow/config_templates/airflow_local_settings.py>`_
```

Comment:
```suggestion
`airflow_local_settings.py template <https://github.com/apache/airflow/blob/|airflow-version|/airflow/config_templates/airflow_local_settings.py>`_
```
Originally it is wrong substitution variable, however variables substitution doesn't work within the links, instead of hardcode to `main` we should change it to correct one (`|airflow-version|`) and add path to generated HTML  (`administration-and-deployment/logging-monitoring/advanced-logging-configuration.html`) into this list

https://github.com/apache/airflow/blob/153fd11f3f360ed1f42c4ac9f1fe08ecf5105f00/docs/conf.py#L333-L337

And after that better to check locally that manual substitution works fine by run [Building the documentation](https://github.com/apache/airflow/blob/main/BREEZE.rst#id29) command, and validate it locally
```
breeze build-docs --package-filter apache-airflow
```

Reply:
The manual subsitution works fine in my local env.

<img width="554" alt="image" src="https://github.com/apache/airflow/assets/39219120/a232d13f-315e-471e-980a-fbadd55ce644">